packages/gafl-webapp-service/src/processors/find-permit.js

```diff
@@ -1,15 +1,15 @@
-import filterPermits from '../../processors/filter-permits.js'
+import filterPermits from './filter-permits.js'
 import crypto from 'crypto'
 
-export default async (permission, request) => {
+export const findPermit = async (permission, request) => {
   /*
    * To stop repeated reads of the API with users repeatably refreshing the page, the transaction cache stores
    * a hash of itself. If the transaction cache has not changed the permit is not recalculated.
    *
    * The section of the transaction cache subject to the hashing algorithm excludes
    * name, address, or anything not effecting permit filter
    */
-  const hashOperand = (({ hash, permit, licensee, ...p }) => p)(permission)
+  const hashOperand = (({ _hash, _permit, _licensee, ...p }) => p)(permission)
```

Comment:
By doing this, you've made it so that the hash, permit and licensee aren't stripped from the object. If you want to maintain the _ prefix notation for unused variables, you'd have to do:
```
const hashOperand = (({ hash: _hash, permit: _permit, licensee: _licensee, ...p }) => p)(permission)
```
The original version is a bit more concise, although potentially less clear. I'll leave it up to you which course to take, I think it can be argued either way.
Have a look here: https://codepen.io/jaucourt/pen/poKGmzY?editors=0012

Reply:
Oops, my bad! Thanks for clarifying. I've opted for longer but more readable as clearly I missed what was going on the first time :)